openssl/src/cms.rs

```diff
@@ -161,4 +303,24 @@ impl CmsContentInfo {
             Ok(CmsContentInfo::from_ptr(cms))
         }
     }
+
+    pub fn partial(
```

Comment:
Why is this called partial if it calls CMS_sign?

Reply:
Documented the method, hopefully it makes more sense now.